### DISCUSSION TITLE
Logseq Plugin: SimpRead Sync

### DISCUSSION THREAD

**Discussion by OverflowCat** [Kenshin/simpread#4220] [服务]
![Logseq Sync](https://user-images.githubusercontent.com/81074/176084816-81304cde-4e3e-4522-93fe-78a08eda0db8.png)

# 介绍

[Logseq SimpRead Sync](https://github.com/Kenshin/simpread/discussions/4117) 已上架到 Logseq 插件中心。

# 前置知识

插件需要配合 https://github.com/Kenshin/simpread/discussions/4117 实现。

# 视频演示

此插件主要将 #4117 手动在侧栏打开标注页的功能变为自动化，并且在左侧编辑区点击链接可以直接在侧栏打开，详情请看下面的视频。

https://user-images.githubusercontent.com/81074/180695552-ba0a5896-891c-43c4-9569-6318daebb1b1.mp4

# 功能简介

插件会在右侧 sidebar 增加一个简悦的选项，打开后是稍后读列表。再次点击按钮可以刷新页面，回到首页顶部。

插件会在工具栏增加一个打开 sidebar 中稍后读列表的快捷入口。图标会根据主题自动变色，不过有些主题使用不同的 CSS Selector，可能会有问题（比如 Bear 主题）。

插件会劫持所有简悦相关链接。因此，打开导出的 `@annote` 页面后，可以实现在 sidebar 标注，笔记内容实时更新。笔记更新为单向，任何在 Logseq 中的修改都会被简悦端覆盖掉。使用此项功能时，需配置好高级账户、同步助手、知识库和增强导出。

# 安装方式

进入 Logseq 插件查找 `simpread` 安装即可。

![image](https://user-images.githubusercontent.com/81074/180631783-59d9e819-3db8-4157-967a-adba403cfe3c.png)

安装后会在工具栏出现一个简悦的 Icon

![image](https://user-images.githubusercontent.com/81074/180632003-4959aa07-a85e-4c7c-a1ac-9a40869975f0.png)

# 项目地址

此项目为简悦官方授权协作方案。

[OverflowCat/logseq-simpread](https://github.com/OverflowCat/logseq-simpread)

**Comment by Kenshin** [comment]
嗨，各位。

简悦官方针对新用户或不喜欢配置的用户，发布了一个新功能：**Logseq 配置库**。

细节请看下面，几分钟就能配置成完成。

***

![img](https://cdn.jsdelivr.net/gh/23784148/upload-images@main/typora/20221007_1665118121.png)

> 针对新用户的一步到位的 Logseq 配置库。

## 🗂 配置库

[简悦 · 配置库](https://www.yuque.com/kenshin/simpread/ds8zk0) 是简悦官方推出的一套针对新用户的极简配置方案，方便新用户用最快的方式使用简悦的各种高级服务，配置库内置了常用的双链笔记用法，如：Notion、Obsidian、Logseq、Roam Research，同时包含了简悦在阅读模式上的一些常规插件：Live Editor、题图、Safari 阅读模式等。

## 💡 特点

用户仅需要下载对应的配置库，修改里面的 UID 为自己的高级账户 UID，即可实现简悦的各种高级操作。

## 💁‍♂️ 受众

新用户或者不想使用手动配置的用户。

## 📝 如何使用

1️⃣ 下载

2️⃣ 修改为自己的 UID

3️⃣ 将必要的文件夹复制粘贴到相应位置

✅ Done！

## 📦 Logseq 配置库

因为 Logseq 与 Obsidian 在插件 API 的支持度有限，所以并没有轻量级使用方案，根据 Logseq 支持的文件格式区别两种方案。

这两种方案均需要同步助手，几乎无任何配置，支持本地快照，自动同步标注，可在 Logseq 中标注，适合 Logseq + 简悦的重度用户。

### 🔌 功能

1️⃣ 支持在 Logseq 中标注

2️⃣ 加入稍后读，自动保存本地快照

3️⃣ 加入标注时，自动将标注内容导入本地快照和 Logseq

4️⃣ 改变稍后读或标注的元数据时，自动将改动的内容保存到本地快照和 Logseq

### 📖 区别

![img](https://imgs.zhubai.love/66078e55f4764b3eaf1a7c1f7cc64c28.png)

### 📙 Org Mode 版

🔗 [Github](https://github.com/Kenshin/simpread-configs/blob/main/logseq@orgmode/Getting%20Started.md) | [语雀](https://www.yuque.com/kenshin/simpread/mdgp9n)

### 📗 Markdown 版

🔗 [Github](https://github.com/Kenshin/simpread-configs/blob/main/logseq@md/Getting%20Started.md) | [语雀](https://www.yuque.com/kenshin/simpread/tukkeo)
